notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<ul><li>We want to limit the vocabulary as much as possible, as we will see later on that it has a large effect on the number of model parameters. --&gt; We want to limit the vocabulary as much as possible, since it has a large effect on the number of model parameters.</li><li><em>too few</em> vocabulary terms --&gt; <em>too few</em> words</li><li>maybe add a quick sentence to describe what the <code>WordPieceTokenizer</code> is and does (a simple example of a "WordPieceTokenized" word would be great)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>